pkg/controllers/namespaceresource/verifier_controller_test.go

```diff
@@ -142,7 +142,7 @@ func TestVerifierAdd_WithParameters(t *testing.T) {
 func TestVerifierAddOrReplace_PluginNotFound(t *testing.T) {
 	resetVerifierMap()
 	var resource = "invalidplugin"
-	expectedMsg := "plugin not found"
+	expectedMsg := "PLUGIN_NOT_FOUND: Plugin: pluginnotfound not found."
```

Comment:
Curious can this msg be improved? "PLUGIN_NOT_FOUND: Plugin: pluginnotfound not found."From a customer's perspective , i would like to know what is the expected path of the plugin